src/python/turicreate/toolkits/_image_feature_extractor.py

```diff
@@ -140,7 +142,7 @@ def next_batch():
         # Encapsulates the work done by TensorFlow for a single batch.
         def handle_request(batch):
             y = self.model.predict(batch)
-            tf_out = [i[0][0] for i in y]
+            tf_out = [i.flatten() for i in y]
```

Comment:
oh, I was actually asking if this line could just be replaced by `tf_out = i.flatten()` or maybe `tf_out = i.flatten().flatten()`